src/state/actions/index.js

```diff
@@ -1,10 +1,15 @@
 export { SET_PROFILE_ID } from './auth/setProfileId';
 export { SET_IS_AUTHENTICATED } from './auth/setIsAuthenticated';
 export { SET_USER_PROFILE } from './userProfile/setUserProfile';
+export { SET_APPLICATION_PROFILE } from './userProfile/setApplicationProfile';
+export { SET_CURRENTUSER_PROFILE } from './userProfile/setCurrentUser';
+export { SET_APPROVAL_SUCCESS } from './applications/setApplicationApprove';
+export { SET_REJECT_SUCCESS } from './applications/setApplicationReject';
 export { SET_FETCH_START } from './lifecycle/setFetchStart';
 export { SET_FETCH_END } from './lifecycle/setFetchEnd';
 export { SET_FETCH_ERROR } from './errors/setFetchError';
-export { MENTOR_ADD_FAILURE, MENTOR_ADD_SUCCESS } from './mentor';
```

Comment:
Great idea to refractor the Mentor add success and failure into one.

Reply:
Thank you max, it made more since to keep it simplified with fetching errors and success states.